slack-base/src/test/java/com/hubspot/slack/client/models/response/conversations/ConversationsRepliesResponseTest.java

```diff
@@ -33,6 +33,10 @@ public void itDeserializesWithRepliesAndNewFields() throws IOException {
     assertThat(response.getMessages().get(0).getReplyUserIds().get()).isEqualTo(Arrays.asList("U061F7AUR", "U061F7AUR", "U061F7AUR"));
     assertThat(response.getMessages().get(0).getReplyUsersCount().get()).isEqualTo(3);
     assertThat(response.getMessages().get(0).getLatestReplyTimestamp().get()).isEqualTo("1483125339.020269");
+    assertThat(response.getMessages().get(1).getReactions().size()).isEqualTo(2);
+    assertThat(response.getMessages().get(1).getReactions().get(0).getName()).isEqualTo("astonished");
+    assertThat(response.getMessages().get(1).getReactions().get(0).getCount()).isEqualTo(3);
+    assertThat(response.getMessages().get(3).getReactions().get(1).getUsers()).isEqualTo(Arrays.asList("U061F7AUR", "U062F7AUR", "U063F7AUR"));
```

Comment:
Let's either assert that `facepalm` reaction is deserialized correctly here as well or remove it from the JSON fixture.